### PR TITLE
Fix support for instance create and add 'sc s-i list' cmd

### PR DESCRIPTION
**Pull request by duglin**
And more clean-up of the model stuff.

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: k8s/service_controller/model/service_binding.go [hunk 1]
```diff
@@ -12,9 +12,11 @@ type CreateServiceBindingResponse struct {
 	Credentials *interface{} `json:"credentials"`
 }
 
+/* Old
```
Comment (MHBauer) [inline k8s/service_controller/model/service_binding.go]: just delete this

File: k8s/service_controller/server/controller.go [hunk 2]
```diff
@@ -244,37 +260,41 @@ func (c *Controller) CreateServiceInstance(w http.ResponseWriter, r *http.Reques
...
+	defer resp.Body.Close()
 	if err != nil {
-		fmt.Printf("Failed to PUT: %#v\n", err)
+		err = fmt.Errorf("Failed to PUT: %#v\n", err)
 		utils.WriteResponse(w, 400, err)
 		return
 	}
-	defer resp.Body.Close()
 
 	// TODO: Align this with the actual response model.
```
Comment (MHBauer) [inline k8s/service_controller/server/controller.go]: is it aligned yet?

**Comment by MHBauer** [comment]
lgtm

looks like I need to start on implementing the 'instancer' interface in the k8s storage backend.

File: k8s/service_controller/server/controller.go (hunk 2 above)
Comment (duglin) [inline k8s/service_controller/server/controller.go]: fixed

File: k8s/service_controller/model/service_binding.go (hunk 1 above)
Comment (duglin) [inline k8s/service_controller/model/service_binding.go]: gone

**Comment by duglin** [comment]
LGTM

**Pull request merged**